src/SFA.DAS.EmployerIncentives.Api/Controllers/LegalEntityController.cs

```diff
@@ -23,5 +24,13 @@ public async Task<IActionResult> RefreshVendorRegistrationFormStatus(DateTime fr
 
             return NoContent();
         }
+
+        [HttpPut("legalentities/{hashedLegalEntityId}/employervendorid")]
+        public async Task<IActionResult> GetAndAddEmployerVendorId(string hashedLegalEntityId)
```

Comment:
GetAndAdd?  But it doesn't get anything?

Reply:
It's getting the VendorId from BC (Vrf) and adding it the LegalEntity in Employer Incentives.

Reply:
Again I can call it AddEmployerVendorIdForLegalEntity("XXX")

Reply:
changed name to AddEmployerVendorIdToLegalEntity()